environment

```diff
@@ -101,6 +101,10 @@ GOOGLE_APPLICATION_SECRETS_SECRETS_NAME="application_secrets.json"
 ES_ALLOWED_SOURCE_IP_SECRETS_NAME="es_source_ip"
 DSS_DEBUG=0
 
+# Enable audit logs by setting ENABLE_AUDIT_LOGS to 1, and specifying the name of the bucket to store logs in.
```

Comment:
Should also include the warning from earlier: `Do not enable audit logs until after you have succesfully deployed the DSS at least once otherwise it will fail`.

Reply:
Fixed